script/openqa-clone-custom-git-refspec

```diff
@@ -55,6 +53,22 @@ extract_urls_from_pr() {
     fi
 }
 
+config_file="$HOME/.config/openqa/github-token.conf"
```

Comment:
Please respect `OPENQA_CONFIG` here which is where all other files are stored. See [Customize configuration directory](https://open.qa/docs/#_customize_configuration_directory).

```suggestion
config_file="${OPENQA_CONFIG}/github-token.conf"
```